test/framework/releaseVersions.go

```diff
@@ -153,3 +139,21 @@ func prodReleases() (release *releasev1alpha1.Release, err error) {
 	}
 	return releases, nil
 }
+
+func getLatestPrevMinorRelease(releases *releasev1alpha1.Release, releaseBranchVersion *semver.Version, targetRelease *releasev1alpha1.EksARelease) (*releasev1alpha1.EksARelease, error) {
```

Comment:
why pass `targetRelease` as an argument and then return it as well?
I would vote to remover the assignment in this method and just return it